docs/Intro_to_TDT.md

```diff
@@ -0,0 +1,160 @@
+# Intro to Taxonomy Development Tools
+
+## Background
+
+Much cell type/class annotation consists simply of names attached to
+individual cells in a cell-by-gene-matrix. Within the Brain Initiative,
+it has become standard to extend this simple annotation by associating
+it with additional information including: hierarchical relationships
+between annotations; synonyms; annotation provenance (who annotated
+what); supporting references; annotation transfer from other taxonomies;
+marker genes; and cell type location, based on dissection and spatial
+transcriptomics (e.g. MerFish).
+
+Extended annotations have been published as standardized "taxonomies"
+following the common cell type nomenclature standard [<u>(Miller et al.
+2020)</u>](https://paperpile.com/c/od1Oz0/WFDT) and as unstandardised
+supplementary material {add figure showing examples with refs}. These
+are published separately from the cell-by-gene-matrices to which they
+refer, with the only link between the two being the cell type/class name
+used in annotation (sometimes supplemented by a file mapping cluster to
+cell IDs). These links break easily (if names are changed) and without a
+standard for versioning, taxonomies published in papers often do not
+completely match cell-by-gene-matrices for the same paper published on
+portals like CZ CELLxGENE. It is also difficult for the community to
+make use of these taxonomy documents in combination with matrix files.
+
+A typical workflow within BICAN starts from an agreed set of clusters.
+Hierarchical annotations (taxonomies) are built on top of these clusters
+by combining them to form a hierarchy of nested cell sets with a fixed
+number of levels (see figure {hierarchical annotation}).
+
+<img src="images/screenshots/media/image1.png"
```

Comment:
relative paths won't work when the doc published at GitHub.io https://brain-bican.github.io/taxonomy-development-tools

we can add `https://raw.githubusercontent.com/brain-bican/taxonomy-development-tools/main/docs/` prefix to all